page_shield_scripts.go

```diff
@@ -0,0 +1,107 @@
+package cloudflare
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/goccy/go-json"
+)
+
+// PageShieldScript represents a Page Shield script.
+type PageShieldScript struct {
+	AddedAt                 string   `json:"added_at"`
+	DomainReportedMalicious *bool    `json:"domain_reported_malicious,omitempty"`
+	FetchedAt               string   `json:"fetched_at"`
+	FirstPageURL            string   `json:"first_page_url"`
+	FirstSeenAt             string   `json:"first_seen_at"`
+	Hash                    string   `json:"hash"`
+	Host                    string   `json:"host"`
+	ID                      string   `json:"id"`
+	JSIntegrityScore        int      `json:"js_integrity_score"`
+	LastSeenAt              string   `json:"last_seen_at"`
+	PageURLs                []string `json:"page_urls"`
+	URL                     string   `json:"url"`
+	URLContainsCdnCgiPath   *bool    `json:"url_contains_cdn_cgi_path,omitempty"`
+}
+
+// ListPageShieldScriptsParams represents a PageShield Script request parameters.
+//
+// API reference: https://developers.cloudflare.com/api/operations/page-shield-list-page-shield-scripts#Query-Parameters
+type ListPageShieldScriptsParams struct {
+	Direction           string `json:"direction"`
```

Comment:
similar to the above on query parameters.

Reply:
same here: url.